cucumber/kubernetes/features/sni.feature

```diff
@@ -0,0 +1,12 @@
+Feature: A permitted Conjur host can authenticate with a valid resource restrictions
+  that is defined in the id and the kubernetes host has a corresponding SSL certificate
+
+  @sni_fails
+  Scenario: Authenticate as a Pod.
+    When I authenticate with authn-k8s as "pod/inventory-pod" without cert and key
+    Then the HTTP status is "401"
+
+  @sni_success
+  Scenario: Authenticate as a Pod.
+    Given I can login to pod matching "app=inventory-pod" to authn-k8s as "*/*"
+    Then I can authenticate pod matching "pod/inventory-pod" with authn-k8s as "*/*"
```

Comment:
I think the unit tests provide amazing validation and coverage for supporting SNI. I'm curious why we are adding a cucumber test which to me seems to be testing the same thing as the unit tests but paying a significantly higher price i.e. deploying a whole Conjur master for each one of these tests ?

Reply:
these are just to validate that the entire solution works with SNI - purely for validation.